test/ApiSpec.scala

```diff
@@ -87,22 +87,27 @@ object ApiSpec extends PlaySpecification with Results {
     override def toMarkerMap: Map[String, Any] = jsonFields
   }
 
+  val defaultItems = Seq(
+    TestItem("arn", "name", "eu-west-1", Map("stage" -> "PROD"), flag=true),
+    TestItem("arn", "name", "eu-west-1", Map("stage" -> "PROD"), flag=false),
+    TestItem("arn", "name", "eu-west-2", Map("stage" -> "CODE"), flag=true)
+  )
 
-  class TestCollectorAgent extends CollectorAgentTrait[TestItem] {
+  class TestCollectorAgent(items: Seq[TestItem] = defaultItems) extends CollectorAgentTrait[TestItem] {
     private val resourceType = ResourceType("test")
     private val label = Label(resourceType, new TestOrigin, 1)
 
-    def get(collector: Collector[TestItem]): Datum[TestItem] = Datum(label, Seq(TestItem("arn", "name", "region")))
+    def get(collector: Collector[TestItem]): Datum[TestItem] = ???
```

Comment:
Not sure if these need an implementation as we're in tests here... If we do not need an implementation, can we add a comment to explain why?